def try_me():
    return "'Try me' was successful!"